modules/network-graph-20-view.adoc

```diff
@@ -0,0 +1,69 @@
+* External entities: These represent entities that are connected outside of your cluster. For more information, see "External entities and connections."
+* Network policies: You can view existing policies for a selected component or view components that have no policies.
+* Network flows: You can select one of the following flows for the graph:
+** Active traffic (default): Selecting this option shows shows observed traffic, focused on the namespace or specific deployment that you selected. You can select the time period for which to display information.
+** Inactive flows: Selecting this option shows potential flows allowed by your network policies, helping you identify missing network policies needed to achieve tighter isolation. You can select the time period for which to display information.
+
+You can also simulate network policies from the network graph view. See "Simulating network policies" for more information.
+
+[discrete]
+== Navigation and user interface in Network Graph (2.0 preview)
+
+* Clicking on items or connections between items in the graph allows you to view additional information about components or connections, or perform actions such as adding a network flow to your baseline. You can also click on the connections between items to view information about the connection.
+* The legend provides information about the symbols in use and their meaning. The legend shows explanatory text for symbols representing namespaces, deployments, and connections on the network graph.
+* You can select additional display options from the drop-down list to display icons such as the network policy status badge, active external traffic badge, and port and protocol labels for edge connections.
+* When {product-title-short} detects changes in network traffic, such as nodes joining or leaving, the network graph displays a notification showing the number of updates available. To avoid interrupting your focus, the graph is not updated automatically. Click the notification to update the graph.
+[discrete]
+== External entities and connections in Network Graph (2.0 preview)
+
+The network graph view shows network connections between managed clusters and external sources.
+In addition, {product-title-short} automatically discovers and highlights public Classless Inter-Domain Routing (CIDR) address blocks, such as Google Cloud, AWS, Azure, Oracle Cloud, and Cloudflare.
+Using this information, you can identify deployments with active external connections and determine if they are making or receiving unauthorized connections from outside your network.
+
+By default, the external connections point to a common *External Entities* box and different CIDR address blocks in the *Network Graph* view.
+However, you can choose not to show auto-discovered CIDR blocks by clicking *Manage CIDR blocks* and deselecting *Auto-discovered CIDR blocks*.
+
+##This section below needs review - is all of it still applicable to Network Graph version 2.0?##
```

Comment:
I'm not entirely sure if they are. @vjwilson do you know? I don't think I can tag Boaz here